pkgs/games/MarbleMarcher/default.nix

```diff
@@ -0,0 +1,97 @@
+{ lib, stdenv, fetchFromGitHub, runtimeShell, cmake, eigen, sfml, libglvnd }:
+
+let
+  pname = "MarbleMarcher";
+
+  # The game doesn’t have any versions in GitHub.
+  # Just taking a commit from the top of the “master” branch.
+  version = "git-2019-04-22";
+
+  src = fetchFromGitHub {
+    owner = "HackerPoet";
+    repo = pname;
+    rev = "478afb8ea9c894d6d8b961583aa557e2f63d6198";
+    sha256 = "1kpn9wnf14wm7867dp3lhb10065h4vzy101gab18zkphlwl1dbwq";
+  };
+
+  esc = lib.escapeShellArg;
+
+  assets = stdenv.mkDerivation {
+    pname = "${pname}-assets-pack";
+    inherit version src;
+
+    phases = [ "installPhase" ];
+
+    installPhase = ''
+      set -eu || exit
+      mkdir -p -- "$out"/assets
+      cp -- "$src"/assets/* "$out"/assets
+    '';
+
+    meta = with lib; {
+      homepage = "https://codeparade.itch.io/marblemarcher";
+      description = "Assets for MarbleMarcher videogame";
+      maintainers = with maintainers; [ unclechu ];
+      platforms = platforms.linux;
+
+      # This assets pack contains:
+      # - A couple of fonts (CC-BY-SA 3.0 and SIL OFL 1.1, with license text files)
+      # - Vertex and fragment shaders under GNU/GPLv2 and later
+      # - Images (png, ico) with not clear license
+      # - Audio files (wav, ogg) with not clear license
+      license = licenses.unfree;
+    };
+  };
+
+  unwrappedName = "${pname}-unwrapped";
+
+in stdenv.mkDerivation {
+  inherit pname version src;
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ eigen sfml libglvnd ];
+
+  phases = [ "buildPhase" "installPhase" ];
+
+  buildPhase = ''
+    set -eu || exit
+    mkdir build
+    (cd build && cmake "$src")
+    cmake --build build
+
+    ( # Making a wrapper that binds assets with the executable
+      printf '#! %s\n' ${esc runtimeShell}
+      printf 'cd -- %s\n' ${esc (esc assets)}
+      printf 'exec -a "$0" %s/bin/%s "$@"\n' "'$out'" ${esc (esc unwrappedName)}
+    ) > ${esc pname}
+
+    chmod +x -- ${esc pname}
+  '';
+
+  installPhase = ''
+    set -eu || exit
+    mkdir -p -- "$out"/bin
+    cp -- build/${esc pname} "$out"/bin/${esc unwrappedName}
+    cp -- ${esc pname} "$out"/bin
+  '';
+
+  meta = with lib; {
+    homepage = "https://codeparade.itch.io/marblemarcher";
+    description = "A videogame based on fractal physics engine";
+
+    longDescription = ''
+      Marble Marcher is a video game demo that uses a fractal physics engine and
+      fully procedural rendering to produce beautiful and unique gameplay unlike
+      anything you've seen before.
+
+      The goal of the game is to reach the flag as quickly as possible. But be
+      careful not to fall off the level or get crushed by the fractal! There are
+      24 levels to unlock.
+    '';
+
+    maintainers = with maintainers; [ unclechu ];
```

Comment:
```suggestion
    '';
    maintainers = with maintainers; [ unclechu ];
```

Reply:
This is a matter of preference. I intentionally separate multi-line blocks with an empty lines around to improve readability. I passed this module through `nixfmt` and this stayed unchanged. Is this change necessary?

Reply:
@SuperSandro2000 I changed this anyway. So this is only an open question from my side.

Reply:
>  I passed this module through nixfmt and this stayed unchanged.

nixfmt does not change whitespace.